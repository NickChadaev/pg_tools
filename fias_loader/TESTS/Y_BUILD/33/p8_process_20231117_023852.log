2023-11-17 02:38:52 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_33 postgres /home/n.chadaev@abrr.local/Y_BUILD/33/stage_gar_c_33.csv 2023-11-13
--------------------------------------------------------
[02:38:52] ... Владимирская обл., id_region: 34, Execute: /home/n.chadaev@abrr.local/Y_BUILD/33/stage_gar_c_33.csv, process_id:25345
[02:38:52] ... Start process
[02:38:52] ...  -- Version
[02:38:52] ...  -- Очистка от данных
[02:38:53] ...  -- Убираем индексы
[02:38:53] ...  -- Set UNLOGGED
[02:38:53] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[02:38:53] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[02:38:53] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[02:38:53] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[02:38:53] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[02:38:53] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[02:38:53] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[02:38:53] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[02:38:54] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[02:38:54] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[02:38:54] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_REESTR_OBJECTS_20231113_55c5a912-7602-45d7-a17b-4d8a47a6de77.XML
[02:53:01] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_ADDR_OBJ_20231113_03086833-9d11-480e-9545-ebadce0e217a.XML
[02:53:22] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_HOUSES_20231113_06978636-0354-4415-9ccb-ea8433bff864.XML
[03:04:41] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_STEADS_20231113_8fdf6e90-745c-40c5-bb68-a7ca2ed247f2.XML
[03:06:17] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_APARTMENTS_20231113_fff6d6da-4820-4f27-bebf-b0c17b1ca226.XML
[03:13:29] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_ROOMS_20231113_d722e176-3b3f-47b8-b940-6fc00fdbdcc1.XML
[03:13:32] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_CARPLACES_20231113_26ce3cac-fa8c-49dc-b86c-4bb06de580d1.XML
[03:13:32] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_ADM_HIERARCHY_20231113_a0a4a894-519d-43b4-83af-a117a2357fb6.XML
[03:29:07] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_MUN_HIERARCHY_20231113_eda68654-7edc-41c7-859d-5fd3968dd57c.XML
[03:43:17] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_CHANGE_HISTORY_20231113_e2095daf-3963-4add-88e5-b4e0ea2cc912.XML
[04:34:11] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_ADDR_OBJ_DIVISION_20231113_5fb7721f-bcb5-471f-8a8a-25b3e2941805.XML
[04:34:11] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_ADDR_OBJ_PARAMS_20231113_b60884ce-af75-4b72-b3c7-5b92fbcc32f9.XML
[04:38:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_STEADS_PARAMS_20231113_70bcf410-f591-4b2f-ae52-67b77db3f0ab.XML
[05:00:39] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_HOUSES_PARAMS_20231113_e0051d2e-089f-474e-b60c-5a3dc8d75972.XML
[06:29:16] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_APARTMENTS_PARAMS_20231113_0092b626-7b9e-4416-a67e-5af0ffc23f3b.XML
[06:43:06] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_ROOMS_PARAMS_20231113_0a59778d-0398-4aad-8d90-572e210e98b3.XML
[06:43:10] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_CARPLACES_PARAMS_20231113_b6c3e33a-8c9b-4c46-8794-d3f4230daa40.XML
[06:43:10] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/33/AS_NORMATIVE_DOCS_20231113_48c6f73e-7075-4b90-8045-811423ae25e9.XML
[06:45:23] ...  -- Data finish
[06:45:23] ...  -- Set LOGGED
[06:45:57] ... Stop process
--------------------------------------------------------
