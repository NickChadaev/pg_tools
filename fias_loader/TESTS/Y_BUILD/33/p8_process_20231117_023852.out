 save_gar_version 
------------------
               82
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                82
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 1156167
gar_fias.as_addr_obj: 22888
gar_fias.as_houses: 843852
gar_fias.as_steads: 128001
gar_fias.as_apartments: 568193
gar_fias.as_rooms: 3594
gar_fias.as_carplaces: 830
gar_fias.as_adm_hierarchy: 1233645
gar_fias.as_mun_hierarchy: 1198161
gar_fias.as_change_history: 4298651
gar_fias.as_addr_obj_division: 9
gar_fias.as_addr_obj_params: 325975
gar_fias.as_steads_params: 1772213
gar_fias.as_houses_params: 7197240
gar_fias.as_apartments_params: 1097406
gar_fias.as_rooms_params: 5701
gar_fias.as_carplaces_params: 157
gar_fias.as_normative_docs: 167370
